flink/src/main/java/org/apache/iceberg/flink/source/DataIterator.java

```diff
@@ -71,8 +71,7 @@
 
   InputFile getInputFile(FileScanTask task) {
     Preconditions.checkArgument(!task.isDataTask(), "Invalid task type");
-
-    return inputFiles.get(task.file().path().toString());
+    return getInputFile(task.file().path().toString());
```

Comment:
Looks like this doesn't need to change. Can you revert this?

Reply:
Sure.